lib/galaxy/tools/sort_collection_list.xml

```diff
@@ -22,7 +22,7 @@
         </conditional>
     </inputs>
     <outputs>
-        <collection name="output" format_source="input" type="list" label="${on_string} (sorted)" >
+        <collection name="output" format_source="input" type_source="input" label="${on_string} (sorted)" >
         </collection>
     </outputs>
     <tests>
```

Comment:
Can you add a test case to the follow block - checkout flatten_collection.xml in the same file for some ideas about syntax and such?

Reply:
Done.

Reply:
Done.